src/operations/front-office/car-search/components/car-model.tsx

```diff
@@ -0,0 +1,87 @@
+import { Typography, Box } from "@mui/material";
+import Image from "next/image";
+import golfImage from "@/assets/images/golf.png";
+import { CarImageList } from "../../home/components";
+
+interface Car {
```

Comment:
I don't know, just because it's called "typescript" not "interfacescript"  use "type" instead

And the type shpuld be taken from the gen client (not now as it's not ready yet, but for the moment, use "type")